.github/workflows/jira-label.yml

```diff
@@ -1,13 +1,23 @@
 name: Create Jira issue
 
+# runs: whenever a label is added to an issue
+# if: the issue title does not yet contain '[API-' and
+#     the label is 'to-jira' or 'Jira'
+# does: create corresponding Jira issue and link the two issues
+
 on:
   issues:
     types: labeled
 
 jobs:
   jira:
-    if: ${{ github.event.label.name == 'to-jira' }}
     runs-on: ubuntu-latest
+    if: |
+      !contains(github.event.issue.title, '[API-') &&
```

Comment:
Brackets are actually not necessary. So the title might include (if added manually) Jira issue key without brackets. Maybe we can consider removing the starting bracket.

Reply:
Makes sense, removed the test for the bracket.